File: src/main/java/site/gachontable/domain/admin/port/in/ExitUser.java
package site.gachontable.domain.admin.port.in;

import site.gachontable.presentation.admin.dto.request.ExitUserRequest;
import site.gachontable.infra.security.principal.AuthDetails;

public interface ExitUser {

    String execute(AuthDetails authDetails, ExitUserRequest request);
}

File: src/main/java/site/gachontable/domain/admin/port/in/ExitUser.kt
package site.gachontable.domain.admin.port.`in`

import site.gachontable.infra.security.principal.AuthDetails
import site.gachontable.presentation.admin.dto.request.ExitUserRequest

interface ExitUser {
    fun execute(authDetails: AuthDetails, request: ExitUserRequest): String
}
